lib/command.js

```diff
@@ -1,10 +1,43 @@
 // shared variables
 import process from 'process';
+import crypto from 'crypto';
 
 // the current speaker queue
 const speakerQueue = [];
 
-export async function processCommand({nick, message, participants, say}) {
+function nonCryptographicHash8(data) {
+  return crypto.createHash('md5').update(data).digest('hex').slice(0, 8);
+}
+
+function getPoll(args, polls, say) {
+  let pollNum;
+  if(args.length > 0) {
+    pollNum = args.shift().replace(/^#/, '');
+    if(isNaN(pollNum)) {
+      say('Expected poll number');
+      return;
+    }
+  } else {
+    if(polls.count === 0) {
+      say('No polls.');
+      return;
+    }
+    pollNum = polls.count;
+  }
+  if(args.length > 0) {
+    say('Unexpected data after poll number')
```

Comment:
In this case the command can (maybe) be corrected by removing the unexpected trailing arguments. To output the whole (modified) command would not be straightforward, because the command/arguments are parsed mutably (using `shift`). But the unrecognized part can be highlighted. How's this? bc189e3aaf425c12806f2b03be764c14e525cb76
```
22:06 <@cel> cgbot-cel: vote 1 on 4 a
22:06 <cgbot-cel> Unexpected data after poll number. Try again without this part: "a"
```

```
22:07 <@cel> cgbot-cel: poll 1 2
22:07 <cgbot-cel> Unexpected data after poll number. Try again without this part: "2"
```